.npmrc

```diff
@@ -0,0 +1 @@
+tag-version-prefix=""
```

Comment:
Could it take it as CLI arguments? so we will not have to add another file (we are removing one gruntfile.js to guplfile.js+ .npmrc)